homeassistant/components/huawei_lte/switch.py

```diff
@@ -0,0 +1,73 @@
+"""Support for Huawei LTE switches."""
```

Comment:
Please remove this platform for now. It should be a separate PR.

Reply:
Done.